src/picai_prep/dcm2mha.py

```diff
@@ -548,7 +544,7 @@ def _filter_localizer_slices(dicom_slice_paths: List[str]) -> List[str]:
                 filtered_dicom_slice_paths.append(path)
         return filtered_dicom_slice_paths
 
-    def _update_dicom_list(self, path: Optional[PathLike] = None, filter_localizer_slices: bool = False):
+    def _update_dicom_list(self, path: Optional[PathLike] = None):
```

Comment:
Functionally a `_reset_dicom_list`, as you are not explicitly updating an existing list, but overwriting it with a new one

Reply:
True it (re)sets the DICOM list. Renamed it to `_set_dicom_list`.